src/pages/OnboardEngagement/ExpensifyClassicPage.tsx

```diff
@@ -0,0 +1,83 @@
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import Navigation from '@libs/Navigation/Navigation';
+import variables from '@styles/variables';
+import ROUTES from '@src/ROUTES';
+
+function ToExpensifyClassicModal() {
+    const StyleUtils = useStyleUtils();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {isExtraSmallScreenHeight, windowHeight} = useWindowDimensions();
+    const canUseTouchScreen = DeviceCapabilities.canUseTouchScreen();
+    const theme = useTheme();
+    const iconWrapperHeight = windowHeight / variables.oldDotWireframeIconWrapperHeightFactor;
+
+    const navigateBack = () => {
+        Navigation.goBack(ROUTES.ONBOARD_MANAGE_EXPENSES);
+    };
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            testID={ToExpensifyClassicModal.displayName}
+        >
+            <View style={[styles.flex1]}>
+                <View style={[StyleUtils.getBackgroundColorStyle(theme.success), StyleUtils.getHeight(iconWrapperHeight), styles.alignItemsCenter, styles.justifyContentCenter, styles.mb4]}>
+                    <HeaderWithBackButton
+                        title="Expensify Classic"
+                        shouldOverlay
+                        shouldShowBackButton
+                        onBackButtonPress={navigateBack}
+                        iconFill={theme.iconColorfulBackground}
+                    />
+                    <View style={styles.pt10}>
+                        <Icon
+                            src={Expensicons.OldDotWireframe}
+                            width={variables.oldDotWireframeIconWidth}
+                            height={variables.oldDotWireframeIconHeight}
+                        />
+                    </View>
+                </View>
+
+                <ScrollView contentContainerStyle={[styles.flex1, styles.ph5]}>
+                    <View>
+                        <Text
+                            style={[styles.textHeadline, styles.preWrap, styles.mb2]}
+                            numberOfLines={2}
+                        >
+                            {translate('expensifyClassic.title')}
+                        </Text>
+                        <Text style={[styles.mb4]}>{translate('expensifyClassic.firstDescription')}</Text>
+                        <Text>{translate('expensifyClassic.secondDescription')}</Text>
+                    </View>
+                </ScrollView>
+                <View style={[styles.ph5, styles.pv4]}>
+                    <Button
+                        success
+                        medium={isExtraSmallScreenHeight}
+                        style={[canUseTouchScreen ? styles.mt5 : styles.mt3, styles.w100]}
+                        text={translate('expensifyClassic.buttonText')}
+                        onPress={() => {}}
```

Comment:
For this part we are going to want to do something like:

```js
Link.openOldDotLink(CONST.OLDDOT_URLS.INBOX);
```

But we need to pass this URL query param:

`?dismissedReason=missingFeatures`

Reply:
Hi Marc, I'm unclear on what to do in this part. 

> But we need to pass this URL query param:

 > `?dismissedReason=missingFeatures`

Reply:
Check [last commit](https://github.com/Expensify/App/pull/35019/commits/67a95b685c2a6217ac8e78c297b115a7872de01e) if my change made sense.